script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_4_1A.txt

```diff
@@ -35,21 +35,21 @@ After hearing her say that much, I understand what she is getting at.
 So that means...
 <Page11>
 "Right. When I said vampire, I was referring to that case.
-You keep telling me to kill you, but so far the only person I want to kill is that person.
-Wouldn't you say that this serial killer is your enemy as well?"
+You keep telling me to kill you, but currently, the only person I want to kill is that person.
+Wouldn't you say that this serial killer is your enemy too?"
 <Page12>
 "――――――"
 My thoughts are racing as I try to catch up with the updated situation.
-I understand the context. I can understand what the girl is saying. But fundamentally, my common sense has been left behind.
+I understand the context. I can understand what the woman is saying. But fundamentally, my common sense has been left behind.
 <Page13>
-"Really, humans are so inefficient, I don't understand it.
+"Really, humans are so inefficient, I truly don't understand them.
 You people call this the work of a 'vampire', yet no one is doing anything about it. It's like a squirrel that won't run away when the forest is on fire."
 <Page14>
 "Well, I mean, there's no such thing as vampires, right?"
 "Mmph."
-The girl frowns sullenly.
+The woman gives a sullen frown.
```

Comment:
I would revert this. I don't feel that one 'gives' a frown, you just frown as a verb.